### PR TITLE
Split up the `ExpressionInput` class into `Check` and `Evaluate`

### PR DESCRIPTION
**Pull request by bjornbugge**
Close issue #3

### CODE REVIEW

File: lib/DeonData.ts [hunk 1]
```diff
@@ -21,10 +21,15 @@ export interface InstantiationOutput {
...
 }
 
-export interface ExpressionInput {
+export interface CheckExpressionInput {
   csl: string;
 }
 
+export interface EvaluateExpressionInput {
+  csl: string;
+  values: Value[];
```
Comment (zoren) [inline lib/DeonData.ts]: Could we make values optional to make this a non-breaking change?

**Comment by zoren** [comment]
We should bump the version number in the MR.

File: lib/DeonData.ts (hunk 1 above)
Comment (bjornbugge) [reply lib/DeonData.ts]: Yes -- that way it also corresponds more closely to the expected input at the webservice's endpoint.

File: lib/DeonRestClient.ts [hunk 2]
```diff
@@ -7,7 +7,14 @@ import {
...
   BadRequestError,
 } from './DeonApi';
-import { InstantiationInput, ExpressionInput, DeclarationInput, Event, Tag } from './DeonData';
+import {
+  InstantiationInput,
+  CheckExpressionInput,
+  EvaluateExpressionInput,
+  DeclarationInput,
+  Event,
+  Tag
```
Comment (hansbugge) [inline lib/DeonRestClient.ts]: You need a trailing comma here to make the linter happy.

Comment (bjornbugge) [reply lib/DeonRestClient.ts]: Done.

**Pull request merged**